### PR TITLE
HTML to ipynb for histograms and heatmaps

### PR DESCRIPTION
**Pull request by priyatharsan**
Update for histograms and heatmaps #802

### CODE REVIEW

**Comment by jdamiba** [comment]
Everything looks good. 💃

File: _posts/matplotlib/histogram/histogram_matplotlib.ipynb [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import random\n",
+    "import numpy\n",
+    "import matplotlib.pyplot as plt\n",
+    "import plotly.plotly as py\n",
+    "import plotly.tools as tls\n",
```
Comment (bcdunbar) [inline _posts/matplotlib/histogram/histogram_matplotlib.ipynb]: Likewise in this PR, you'll want to #1048 (comment) in all the notebooks. Also, ensure that it is the same way @cldougl had it i.e. "plotly packages first then a space then additional packages "

Comment (bcdunbar) [reply _posts/matplotlib/histogram/histogram_matplotlib.ipynb]:
```
import plotly.plotly as py
import plotly.tools as tls

import matplotlib.pyplot as plt
import numpy as np
```

File: _posts/matplotlib/histogram/histogram_matplotlib.ipynb [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+    "fig = plt.gcf()\n",
+    "plotly_fig = tls.mpl_to_plotly( fig )\n",
+    "py.iplot(plotly_fig, filename='mpl-basic-histogram')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Multiple datasets\n",
```
Comment (bcdunbar) [inline _posts/matplotlib/histogram/histogram_matplotlib.ipynb]: Capitalize 😄 "Multiple datasets" > "Multiple Datasets"

File: _posts/matplotlib/histogram/histogram_matplotlib.ipynb [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+    "\n",
+    "plotly_fig = tls.mpl_to_plotly( fig )\n",
+    "py.iplot(plotly_fig, filename='mpl-multi-histogram')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Plot two histograms together\n",
```
Comment (bcdunbar) [inline _posts/matplotlib/histogram/histogram_matplotlib.ipynb]: Same here. Capitalize. "Plot Two Histograms Together"

Comment (priyatharsan) [reply _posts/matplotlib/histogram/histogram_matplotlib.ipynb]: Yeah I forgot to update this PR as well

**Comment by bcdunbar** [comment]
Thanks for making the changes 💃

**Pull request merged**